Sources/Testing/Running/Configuration.TestFilter.swift

```diff
@@ -263,13 +326,13 @@ extension Configuration.TestFilter {
   ///   - testGraph: The test graph to filter.
   ///
   /// - Returns: A copy of `testGraph` with filtered tests replaced with `nil`.
-  func apply(to testGraph: Graph<String, Test?>) -> Graph<String, Test?> {
-    var result = _kind.apply(to: testGraph)
+  func apply(to testGraph: Graph<String, Test?>) throws -> Graph<String, Test?> {
+    var result = try Operation(_predicate).apply(to: testGraph)
```

Comment:
Do we ever need to separate `Operation` from `apply(to:)`? If not, can we just absorb the former into `apply(to:)`?

Reply:
You mean e.g. inline `Operation` enum into this function? I considered that but felt it would make the function body quite a bit more complicated. As it stands we're using access control to scope it pretty narrowly

Reply:
We could still split up the implementation into multiple functions/steps, but keep the _interface_ simple.